### ISSUE TITLE
Question about finetuning stereo matching nets on booster datasets.

### ISSUE THREAD

**Issue by Double2016** [CVLAB-Unibo/Depth4ToM-code#1]
As we all kown,booser training datasets have ground truth for stereo matching.Both virtual and merged depth can't be better than gt.  So, how can the Table 4 of your experimental results be reproduced? What kind of distillation loss can work?

**Comment by mattpoggi** [comment]
Hi @Double2016,
not sure I got your point :) if you mean that training on the virtual and merged depths cannot be better than training on real gt, that's true. Indeed, you can have a quantitative comparison by looking at Table 3 in our journal paper (https://arxiv.org/pdf/2301.08245.pdf) and Table 4 in ToM paper.

I'm collecting the two in a screenshot, where I label with colors:
- original RAFT-Stereo (green)
- original CREStereo (light blue)
- fine-tuned RAFT-Stereo (yellow): with GT (left), with our proxies (right)
- fine-tuned CREStereo (purple): with GT (left), with our proxies (right)

<img width="980" alt="Screenshot 2024-03-04 alle 14 41 00" src="https://github.com/CVLAB-Unibo/Depth4ToM-code/assets/5264662/b102b91c-cf65-4abc-834c-e596ba3e5961">

Hope this helps

**Comment by Double2016** [comment]
Thank you very much for your reply,which solved my doubts.
